### PR TITLE
Add GoogleAuthenticator as a composer dependency

### PR DESCRIPTION
**Pull request by therealssj**
As referenced in this https://github.com/d8-contrib-modules/tfa_basic/issues/2
Modules should use composer for dependencies.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "drupal/tfa_basic",
+    "description": "Basic Authentication Plugin for TFA Module",
```
Comment (nerdstein) [inline composer.json:4]: Missing "type": "drupal-module",

**Comment by nerdstein** [comment]
Reviewed and looks good, missing one small thing and I will merge

**Comment by therealssj** [comment]
Fixed!
Thnx for pointing out!

**Comment by nerdstein** [comment]
Merged, thank you!

**Pull request merged**